express/blocks/hero-animation/hero-animation.js

```diff
@@ -231,6 +231,12 @@ export default async function decorate($block) {
         transformToVideoLink($div, videoLink);
       }
       addFreePlanWidget($div.querySelector('.button-container') || $div.children[0]);
+      // stack CTA and free plan widget if CTA text exceeds 24 characters
```

Comment:
I am not fully sure this would be a fully valid rule. JP was also problematic and it is still the case: https://mwpw-116370--express-website--adobe.hlx.page/jp/express/?lighthouse=on

I think the rule is: CTA button should never ellipsis. If it does, move the rest next line.
I have quickly google it and it seems possible to do something like:

```
function isEllipsisActive(el) {
     return (el.offsetWidth < el.scrollWidth);
}
```
to know if an element "ellipsises" or not.

Reply:
Unfortunately it's not so easy, because the button isn't rendered yet. Both `offsetWidth` and `scrollWidth` are 0 at the time we need to decide whether to use a stacked design or not (to avoid CLS). 

Maybe we could use a different trigger point for asian languages, similar to what we are doing for heading auto-resizing...

Reply:
I think we can get away with a special treatment for JP here, which is something we have in many other places, too:
- https://mwpw-116370--express-website--adobe.hlx.page/jp/express

Itested all other languages, and the ideal threshold seems to be 23 characters, which happens to be the exact length of the English version. Only CN and TW stay below, all other translations are longer and will therefore be stacked.